crates/swc_ecma_transforms_module/tests/fixture/common/interop-node/export-from-1/output.cjs

```diff
@@ -0,0 +1,5 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", {
+    value: true
+});
+_exportStar(require("foo"), exports);
```

Comment:
We don't need interopRequireWildcard here?

Reply:
No, it's not necessary.

1. `export *` never reexport `default`.
2. import node-esm from node-esm, everything is as same as browser-esm.
3.  import cjs in node-esm, node will wrap all export ident in `default` <- this is different

The only thing we need special handling is importing cjs to node-esm.
We should wrap everything in `default` to mimic Node's behavior.

```JavaScript
// foo.cjs
exports.foo = function foo() {};

// bar.cjs
exports.bar = function bar() {};

// index.js
import foo from "./foo.cjs";
import * as bar from "./bar.cjs";

export * from "./foo.cjs"
export * from "./bar.cjs";

console.log(foo); 
// { foo: [Function: foo] }

console.log(bar);
// [Module: null prototype] {
//     bar: [Function: bar],
//     default: { bar: [Function: bar] }   <- Node will wrap all exports in `default`
//   }
```

But, we never reexport `default`,  and function bar is in both namespace and default.
it's ok just use `_exportStar` and `require` to reexport everything.

Let's see the Babel‘s transforms.
Input:
```JavaScript
import * as foo from "./foo.cjs"; // trigger `_interopRequireWildcard`

export * from "./foo.cjs"
export * from "./bar.cjs";

foo;
```

output:
```JavaScript
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});

var foo = _interopRequireWildcard(require("./foo.cjs"), true);

Object.keys(foo).forEach(function (key) {
  // omit...
});

var _bar = require("./bar.cjs");

Object.keys(_bar).forEach(function (key) {
// omit...
});

foo;
```

You can see that foo is wrapped by `_interopRequireWildcard` and bar is not, and there is no difference when re-exporting.

Reply:
Thank you for the explanation!